examples/live/make_fit_coeffs.py

```diff
@@ -0,0 +1,21 @@
+import h5py
```

Comment:
Makes files which can be used as the `fit_coeffs` statistic. These are not of any scientific use, but the code will accept them and run properly

Reply:
Could you turn your note into a docstring for this file?